pygmt/src/config.py

```diff
@@ -20,6 +22,15 @@ class config:  # pylint: disable=invalid-name
     Full GMT defaults list at :gmt-docs:`gmt.conf.html`
     """
 
+    # Manually set the __signature__ attribute to enable tab autocompletion
+    _keywords = ["FONT_LABEL", "FONT_TAG", "FONT_TITLE"]
```

Comment:
`gmt defaults` can output all the keywords and their current values. So at least we can call `gmt defaults` and extract the keywords from the output.

Reply:
Calling `gmt defaults` is not a perfect idea, because the code will be executed when importing the `pygmt` package. Perhaps manually maintaining the the long list of keywords is not a bad idea because the list of GMT keywords don't change much between versions.

Reply:
Ok I guess. Even if the list gets outbof date because GMT adds new configs, people can still use it, just without tab completion.